projects/storefrontlib/shared/utils/validators/custom-form-validators.ts

```diff
@@ -58,6 +60,29 @@ export class CustomFormValidators {
       : { cxInvalidPassword: true };
   }
 
+  // TODO: (CXSPA-7567) Remove after removing formErrorsDescriptiveMessages feature toggle
+  /**
+   * Checks control's value with predefined password regexp
+   *
+   * NOTE: Use it as a control validator
+   *
+   * @deprecated Use passwordValidators instead
+   * @static
+   * @param {AbstractControl} control
+   * @returns {(ValidationErrors | null)} Uses 'cxInvalidPassword' validator error
+   * @memberof CustomFormValidators
+   */
+  static strongPasswordValidator(
+    control: AbstractControl
+  ): ValidationErrors | null {
+    const password = control.value as string;
```

Comment:
If the value might not always be a string (the form control value could be null or undefined), it's better to check the type or handle non-string cases gracefully? if this method get call after null check then might be fine.

Reply:
It's the same as every other custom validator. In the case of a password, it should always be a string and will always include a required validator to check for null or undefined values.